dbt/models/proximity/columns.md

```diff
@@ -181,6 +181,21 @@ Nearest golf course ID, either Cook County ID or OSM ID.
 Golf course data sourced from Cook County GIS and OpenStreetMap
 {% enddocs %}
 
+## nearest_permit_dist_ft
+
+{% docs column_nearest_permit_dist_ft %}
+Nearest home improvement exemption distance (feet).
```

Comment:
We can't use Home Improvement Exemptions (HIE) as markers of permits/new construction, they're really an entirely separate thing. See my other comments for alternatives.